pyomo/contrib/mindtpy/config_options.py

```diff
@@ -575,4 +575,23 @@ def check_config(config):
                 'The threads parameter is corrected to 1 since incumbent callback conflicts with multi-threads mode.')
     if config.solution_pool:
         if config.mip_solver not in {'cplex_persistent', 'gurobi_persistent'}:
+            if config.mip_solver in {'appsi_cplex', 'appsi_gurobi'}:
+                config.logger.info("Solution pool does not support APPSI solver yet.")
```

Comment:
Consider removing yet, as it looks like a promise which I do not know if we want to hold.